README.md

```diff
@@ -8,32 +8,51 @@ Appwrite playground is a simple way to explore the Appwrite API & Appwrite Ruby
 
 ## Get Started
 
-This playground doesn't include all of Ruby best practices but rather intended to show some of the most simple examples and use cases of using the Appwrite API in your Ruby application. Rubocop is enforced and the configuration can be found at [.rubocop.yml](.rubocop.yml)
+This playground does not adhere to Ruby best practices but rather is intended to show some of the most simple examples and use cases of using the Appwrite API in your Ruby application. Rubocop is enforced and the configuration can be found at [.rubocop.yml](.rubocop.yml)
 
-## System Requirements 
-* A system with Ruby installed.
-* You have readily available AppWrite running instance (localhost in most cases).
-* Create a project in AppWrite instance using console.
-* Generate a secret key in the AppWrite instance using console.
+## Requirements 
+* A system with Ruby 2.7+ or Docker installed.
```

Comment:
Can we apply same copy from sdk-for-python for requirements and instlaation? I will also try to stay consistent in other playgrounds